react/features/subtitles/middleware.js

```diff
@@ -47,23 +51,46 @@ MiddlewareRegistry.register(store => next => action => {
  */
 function _endpointMessageReceived({ dispatch, getState }, next, action) {
     const json = action.json;
+    const translationLanguage
+        = APP.conference._room.getLocalParticipantProperty(
+            'translation_language');
```

Comment:
can we make `translation_language` a constant instead of having a magic hard-coded value?